week6/assignment2_1/includes/ex.hpp

```diff
@@ -0,0 +1,31 @@
+#ifndef EX_HPP
+#define EX_HPP
+#include <iostream>
+#include <limits>
+
+class Positive_integers {
+    public:
+        int number{0};
+        Positive_integers() = default; 
+        Positive_integers(int num);
+        void print_number();
+        int add(Positive_integers obj);
+        int subtract(Positive_integers obj);
+        int max_number{std::numeric_limits<int>::max()};
+};
+
+class Count_instances {
```

Comment:
Here I would do the same and rename it to `Instance_counter` or smth., since again, that is what you get when you instantiate the type.